setup.py

```diff
@@ -63,7 +63,10 @@
     'flask-shell-ipython>=0.3.0',
     'flask-talisman>=0.3.2',
     'invenio-base>=1.0.0a14,<1.1.0',
+    'invenio-cache>=1.0.0a1',
     'invenio-config>=1.0.0b2,<1.1.0',
+    # FIXME maybe it should be set in `invenio-cache`?
+    'redis>=2.10.5',
```

Comment:
it's a bit tricky....similar to choosing between mysql, postgresql etc....note sure exactly how to do it...